### DISCUSSION TITLE
point_0721

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#287] [Roy]
"20200511T074609_20200511T075431_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147613136-424f36d8-162d-46b9-b6f2-634563fc22d5.png)

"20190611T074611_20190611T075718_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147613236-933c2d97-1369-4900-819b-49ca7cd3f39d.png)

"20190318T074639_20190318T080034_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147613333-be07c1df-6890-424d-835e-92f96202ffd4.png)
![image](https://user-images.githubusercontent.com/54947195/147613378-f5cdfca6-ea33-46e1-923c-d51dcc28ab64.png)

"20190226T074859_20190226T080102_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147613426-f5df3cbd-e0c9-495c-a481-904401afe8b3.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200511T074609_20200511T075431_T37QED"
![Captura](https://user-images.githubusercontent.com/86502862/153615376-cda9ab92-bf9f-4f6f-b3bc-65cc2d6103a4.JPG)
